CHANGELOG.md

```diff
@@ -8,7 +8,41 @@ core section.***
 - ***When possible please provide the number of the pull request(s) containing the 
 changes in the following format: PR #1234***
```

Comment:
should probably remove the text above before release?

Reply:
lines 1 to 9 ?

Reply:
3 to 9, the heading is still fine..